main/GlobalInit.c

```diff
@@ -0,0 +1,20 @@
+
+#include"main.h"
+
+extern QueueHandle_t BufQueue1;
+extern SemaphoreHandle_t GetResponseSemaphore;
+
+
+
+
+
+void GlobalInit()
+{
+    ESP_ERROR_CHECK(nvs_flash_init());
```

Comment:
nvf_flash init in its module that call from mainm, so dont need reinit again in any modules.

Reply:
ok